forge.config.ts

```diff
@@ -9,6 +9,17 @@ import { FusesPlugin } from '@electron-forge/plugin-fuses'
 import { FuseV1Options, FuseVersion } from '@electron/fuses'
 import { getPlatform, getArch } from './src/utils/electron'
 
+let resources = [
+  './resources/' + getPlatform() + '/' + getArch(),
+]
+
+if (getPlatform() === 'mac') {
+  resources = [
+    './resources/mac/arm64',
+    './resources/mac/x86_64',
+  ]
+}
```

Comment:
Nit: I'm not a big fan of reassigning variables, so wanted to suggest an alternative:
```ts
function getPlatformSpecificResources() {
  if (getPlatform() === 'mac') {
    return ['./resources/mac/arm64', './resources/mac/x86_64']
  }

  return ['./resources/' + getPlatform() + '/' + getArch()]
}

const config: ForgeConfig = {
  // ...
  extraResource: [
    // ...
    ...getPlatformSpecificResources()
  ]
}
```